samples/subsys/input/input_dump/boards/stm32u083c_dk.overlay

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2024 Arif Balik <arifbalik@outlook.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&tsc {
+status = "okay";
+
+spread-spectrum;
+spread-spectrum-deviation = <100>;
```

Comment:
it was, but my local compliance_check.py fails when I do

Reply:
how does it fail? maybe you were using spaces instead of tabs? btw I think this could go straight in `boards/st/stm32u083c_dk/stm32u083c_dk`, no point splitting them in the overlay

Reply:
Yes, just checked, it was spaces instead of tabs